journal/202103.md

```diff
@@ -0,0 +1,466 @@
+# {DRAFT} Decred Journal – March 2021
+
+![{alt text}](../img/{file.jpg} "{hover tooltip text}")
+
+_Image: {"$title by $author. $optional-description" for artwork, "$description" for photos}_
+
+Highlights for March: {
+
+- consensus vote
+- v1.6.2 close
+- politeia mostly rewritten for a scalable and flexible storage backend
+- bison thing maybe
+
+}
+
+Contents: { sync at the end }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Integrations](#integrations)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Community Discussions](#community-discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+<a id="dcrd" />
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+Merged in master and the upcoming v1.6.2 release:
+
+- only send [fast block](https://github.com/decred/dcrd/pull/2606) announcements to full node peers, and don't send them to lightweight clients that cannot properly handle them. This significantly improves dcrwallet's SPV operation. { how much better? }
+
+Main addition of the month is the new package for handling [standard addresses](https://github.com/decred/dcrd/pull/2610) that is easier to follow and use than the code it replaces. It is also more generic, allowing to add new address types in the future. If you have ever wondered how "scripts" and "addresses" are related, or how "standard" is different from "consensus", check the shiny new [README](https://github.com/decred/dcrd/tree/master/txscript/stdaddr) of the package or follow the examples and comprehensive tests to fully understand how it works. Thanks to all reviewers and welcome to the newcomers - it's always good to get a fresh pair of eyes!
```

Comment:
It's probably also worth calling out that this, along with the other merged address-related PRs are all part of laying some initial groundwork and infrastructure towards future consensus votes.

Reply:
Thanks, added!

I know that all these changes are not random and are part of a bigger puzzle even when they look like "innocent" package moves, but I didn't expect a CV to be a factor here :)

_You are correct that they are not going to the Moon, they are going to Mars. But you still think too small, bee. These people are not going to Mars to build a cozy little base and settle. First thing they build will be a _starport_ to launch a Titan expedition. Although in the process they might terraform Mars, just for nice views to look at while they design the next thing._